server/routes/caseNotes/confirmation/newCaseNoteConfirmationPresenter.ts

```diff
@@ -0,0 +1,41 @@
+import { SummaryListItem } from '../../../utils/summaryList'
+import PresenterUtils from '../../../utils/presenterUtils'
+import utils from '../../../utils/utils'
+import SentReferral from '../../../models/sentReferral'
+import Intervention from '../../../models/intervention'
+
+export default class NewCaseNoteConfirmationPresenter {
```

Comment:
As mentioned on an earlier PR, do we want to rename this to `AddCaseNote...` for consistency?

Reply:
hmmm, I renamed it 

ah looks like these were included accidently when rebasing (the new renamed classes exist and these old ones aren't being used) I'll update the commit to remove them